src/Parser/RetributionPaladin/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import Icon from 'common/Icon';
+import { formatPercentage, formatDuration } from 'common/format';
+
+import CoreAlwaysBeCastingHealing from 'Parser/Core/Modules/AlwaysBeCastingHealing';
```

Comment:
This can be the regular AlwaysBeCasting module since you won't need the healing behaviors.